robyn/router.py

```diff
@@ -82,22 +83,46 @@ def _format_response(
             )
         return response
 
+    def validate_handler_args(self, handler, endpoint, dependencies):
```

Comment:
we should be using this api to test. why are we using `app.get_injected_dependencies` to test?